src/sage/combinat/matrices/hadamard_matrix.py

```diff
@@ -408,14 +408,14 @@ def T(i, j):
 
     e = matrix([[1] * (2*m)])
     one = matrix([1])
-    H = block_matrix([[ one,       -e,  one,        e,  one,        e,  one,        e],
```

Comment:
Is there not a way to just override the pycodestyle warning for these whitespace issues by something like a #noqa annotation.
Obviously, the code was formatted in this file for clarity; would anyone consider the deformatting to be an improvement?

Reply:
Indeed, these changes make the data much less readable. One wants matrices aligned in the source code.